apps/news/news.arc

```diff
@@ -580,7 +580,7 @@
            (userstyle ,gu)
            (usertheme, gu)
            (gentag link "rel" "icon" "href" "/favicon.ico")
-           (gentag "meta" "name" "viewport" "value" "width=device-width")
+           (gentag "meta" "name" "viewport" "content" "width=device-width, initial-scale=1")
```

Comment:
For some reason this wasn't configured properly? Swapping this out from "value" to "content" allowed this "toggle device toolbar" inspector tool to work at all, which makes testing different device widths _much_ easier. Before the site wouldn't properly respond to different viewports.

![Screenshot_20231023_111826](https://github.com/jgrahamc/twostopbits/assets/318925/49f16857-e2df-4013-928b-fd0a34ad16ae)